core/src/main/resources/static/js/envs.js

```diff
@@ -292,7 +292,7 @@ app.controller("envsCtrl", function($scope, $http, $location, $window) {
                         data: {'clusterType' : 'all'}
                     }).success(function(output) {
                         $scope.allclustersset = output;
-                        if(output[0] != null){
+                        if(output && output.length > 0 && output[0] != null){
                             $scope.resultPages = output[0].allPageNos;
```

Comment:
does output on its own act as a truthy check here? check for non null and not undefined?

Reply:
yes, actually it does work on its own.